.github/workflows/nodejs-publish.yaml

```diff
@@ -18,9 +18,12 @@ jobs:
         run: |
           mvn package -DskipTests
       - name: print auth token
-        run: echo "//registry.npmjs.org/:_authToken=${NPM_AUTH_TOKEN}\n" >> ~/.npmrc
+        run: npm config set registry https://registry.npmjs.org
         env:
           NPM_AUTH_TOKEN: ${{ secrets.NPM_AUTH_TOKEN }}
+      - name: cat npmrc 
+        working-directory: ./javascript/src/typescript
+        run: cat ~/.npmrc
```

Comment:
Why is it displayed ? It should never be the case

Reply:
it was for a sanity check, and it isn't there anymore.  Debugging these is kind of tough because I can't do it on my fork, and it needs to be on master.